packages/roosterjs-editor-api/lib/format/setFontName.ts

```diff
@@ -14,5 +15,9 @@ export default function setFontName(editor: IEditor, fontName: string) {
     // for here, we use CSS font-family style
     applyInlineStyle(editor, (element, isInnerNode) => {
         element.style.fontFamily = isInnerNode ? '' : fontName;
+
+        if (safeInstanceOf(element.parentElement, 'HTMLLIElement')) {
+            setListItemStyle(element.parentElement, ['font-family']);

```

Comment:
If there are multiple elements are involved for this action, this function will be run multiple times, but actually we only need it run once. So you can cache all parent nodes. Then get all distinct LI items and run them one by one.

Reply:
Yes you are right.
I will add an array to store the unique parent nodes. So after we finish applying the styles on the child elements, calling this action only once for each individual Node.